javascript/javascript1/week1/homework_soheib/houseyPricey.js

```diff
@@ -0,0 +1,40 @@
+let peterHouse = {
+    width: 8,
+    depth: 10,
+    height: 10,
+    garden: 100,
+    price: 2500000
+};
+
+let juliaHouse = {
+    width: 5,
+    depth: 11,
+    height: 8,
+    garden: 70,
+    price: 1000000
+};
+
+function volumeCalc(house){
+    let volume = house.depth * house.width * house.height
+    return volume;
+}
+
+function houseValueCalc(house){
+    let houseValue = volumeCalc(house) * 2.5 * 1000 + house.garden * 300;
+    return houseValue
+}
+
+function compareValue(house) {
+    if(houseValueCalc(house) > house.price ) {
+        let diff = Math.floor(houseValueCalc(house)) - house.price; 
+        return console.log(`Ouch! This person is paying ${diff} too much for the house :(`)
+    } else if (houseValueCalc < house.price) {
```

Comment:
What is houseValueCalc without paramathers?

Reply:
🫣

Reply:
I have rewritten it, so that it is more concise and readable. 
I stored the recurring calculations in values (const) and discovered Math.abs  that gives the absolute value, so that I don't need to calculate the difference twice. 
Many thanks for the very valuable insights :D